src/compliment/sources/ns_mappings.clj

```diff
@@ -101,17 +101,36 @@
                                  arglists :function
                                  :else :var)
                      :ns (str (or (:ns var-meta) ns))}
-              (and arglists(:arglists *extra-metadata*))
+              (and arglists (:arglists *extra-metadata*))
               (assoc :arglists (apply list (map pr-str arglists)))
 
               (and doc (:doc *extra-metadata*))
               (assoc :doc (generate-docstring var-meta)))))))))
 
+(defn- resolve-var
+  "Examples:
+  \"map\" => #'clojure.core/map
+
+  ;; Resolves aliases
+  \"some-alias/some-fn\" => #'some.full.ns/some-fn
+
+  ;; Ignores preceding literals (i.e. @, #', ')
+  \"#'map\" => #'clojure.core/map
+  \"@#'some-alias/db\" => #'some.full.ns/db
+  "
+  [symbol-str ns]
+  (let [strip-literals                    #(->> % (re-find #"^[@#']*(.+)") last)
```

Comment:
The code here allows multiple `@` while the code above doesn't. Regardless of which way you want it, I'd rather have it consistent across all code that does this.